test/battle/hold_effect/kee_berry.c

```diff
@@ -0,0 +1,44 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gItems[ITEM_JABOCA_BERRY].holdEffect == HOLD_EFFECT_JABOCA_BERRY);
```

Comment:
This whole file is for jaboca berry, but should be for kee berry.

Reply:
Should be fixed.